dotCMS/src/main/java/com/dotcms/rest/api/v1/page/PageResource.java

```diff
@@ -232,7 +232,8 @@ public Response renderHTMLOnly(@Context final HttpServletRequest request,
                                    @PathParam("uri") final String uri,
                                    @QueryParam("mode") @DefaultValue("LIVE_ADMIN") final String modeStr,
                                    @QueryParam(WebKeys.CMS_PERSONA_PARAMETER) final String personaId,
-                                   @QueryParam("language_id") @DefaultValue("LIVE_ADMIN") final String languageId) {
+                                   @QueryParam("language_id") @DefaultValue("LIVE_ADMIN") final String languageId,
```

Comment:
That `@DefaultValue("LIVE_ADMIN")` looks wrong for a `language_id`, I think it is a typo